api/src/main/resources/marquez/db/migration/V71__add_mat_views_lineage_metrics.sql

```diff
@@ -0,0 +1,23 @@
+CREATE MATERIALIZED VIEW IF NOT EXISTS lineage_metrics_hourly AS
```

Comment:
Using `lineage_metrics_hourly` does seem to imply that the table would have different metrics per row that can be aggregated.

I would suggest we define a more specific name for the view:

```
lineage_events_by_type_hourly_view
```